src/geode/geometry/bounding_box.cpp

```diff
@@ -128,6 +133,59 @@ namespace geode
             return line_intersects( line );
         }
 
+        template < index_t T = dimension >
+        typename std::enable_if< T == 2 || T == 3, bool >::type intersects(
+            const Triangle< T >& triangle ) const;
+
+        template < index_t T = dimension >
+        typename std::enable_if< T == 3, bool >::type intersects(
+            const Tetrahedron& tetra ) const
+        {
+            const auto& vertices = tetra.vertices();
+            return intersects( { vertices[0].get(), vertices[1].get(),
+                       vertices[2].get() } )
+                   || intersects( { vertices[0].get(), vertices[1].get(),
+                       vertices[3].get() } )
+                   || intersects( { vertices[0].get(), vertices[2].get(),
+                       vertices[3].get() } )
+                   || intersects( { vertices[1].get(), vertices[2].get(),
+                       vertices[3].get() } );
+        }
```

Comment:
|| bbox.center() is in the tetrahedron

Reply:
No need. The intersections tests are enough